File: 1704-determine-if-string-halves-are-alike/1704-determine-if-string-halves-are-alike.py
class Solution:
    def halvesAreAlike(self, s: str) -> bool:
        l, r, vo = 0, 0, "aeiouAEIOU"
        for i in range(len(s)):
            if s[i] in vo:
                if i<len(s)//2:
                    l = l + 1
                else:
                    r = r + 1
        return l == r
        

File: 1704-determine-if-string-halves-are-alike/1704-determine-if-string-halves-are-alike.cpp
class Solution {
public:
    bool halvesAreAlike(string s) {
        int l1 = 0, l2 = 0;
        string vo = "aeiouAEIOU";
        for(int i=0; i<s.size(); i++){
            if(count(vo.begin(), vo.end(), s[i])){
                if(i<s.size()/2)  l1++;
                else                    l2++;
            }
        }
        return l1==l2;
    }
};